satellite/console/consoleweb/server.go

```diff
@@ -43,7 +44,7 @@ var Error = errs.Class("satellite console error")
 // Config contains configuration for console web server
 type Config struct {
 	Address         string `help:"server address of the graphql api gateway and frontend app" default:"127.0.0.1:8081"`
-	StaticDir       string `help:"path to static resources" default:""`
+	StaticArchive   string `help:"path to static resources" default:"assets.zip"`
```

Comment:
Does console development workflow still work? It would be inconvenient to use zips for development.

I think we want to allow for using local filesystem as well and not constrain to only zips.

Reply:
I agree with that

Reply:
I agree.